.github/workflows/meshery-cloud-release.yml

```diff
@@ -298,3 +298,4 @@ jobs:
         envs: GIT_VERSION
         script: |
           helm upgrade -f ~/aws-m03/meshery-cloud/install/kubernetes/helm/layer5-cloud/values.yaml layer5 ~/aws-m03/meshery-cloud/install/kubernetes/helm/layer5-cloud/ -n prod-cloud --set=image.tag=production-$GIT_VERSION --set-file 'kratos.kratos.emailTemplates.recovery.valid.subject'=./aws-m03/meshery-cloud/config/email-templates/valid/email-recover-subject.body.gotmpl --set-file 'kratos.kratos.emailTemplates.recovery.valid.body'=./aws-m03/meshery-cloud/config/email-templates/valid/email-recover.body.gotmpl --set-file 'kratos.kratos.emailTemplates.verification.valid.subject'=./aws-m03/meshery-cloud/config/email-templates/valid/email-verify-subject.body.gotmpl --set-file 'kratos.kratos.emailTemplates.verification.valid.body'=./aws-m03/meshery-cloud/config/email-templates/valid/email-verify.body.gotmpl
+          kubectl apply -f ./aws-m03/ingress.yaml
```

Comment:
Good, good. We'll want to track this ingress configuration inside of the meshery-cloud repo under the respective environment folder. It'd be good to ensure this file is pushed to layer5io/meshery-cloud and that the file path here is updated to reflect that location.

Reply:
@leecalcote yes we can once the helm issue is sorted. But for now we need this line in the workflow, otherwise everytime there is a cloud release, the nginx will reset and the cloud-ws setting will be lost. 
So, at the moment this needs to be approved for merger.